src/Catrobat/CatrobatCode/Parser/Bricks/BrickFactory.php

```diff
@@ -362,6 +371,15 @@ public static function generate(SimpleXMLElement $brick_xml_properties)
         case Constants::WRITE_VARIABLE_TO_FILE_BRICK:
             $generated_brick = new WriteVariableToFileBrick($brick_xml_properties);
             break;
+        case Constants::PARAMETERIZED_BRICK:
```

Comment:
please align all cases in this file... I'm wondering that our fixer can't do this automatically...

E.g.
![image](https://user-images.githubusercontent.com/40868718/90858127-d9a75880-e385-11ea-8c18-8d17956853d4.png)